Troubleshooting.md

```diff
@@ -14,7 +14,29 @@ Logs can provide an essential piece of information. In the next few paragraphs,
 
 ### 2.1 BTCPay Logs
 
-Since the v1.0.3.8, you can easily access BTCPay Server logs from the front-end. If you are a server admin, go to Server Settings > Logs and open the logs file. If you don't know what a particular error in the logs means, make sure to mention it when troubleshooting.
+Since the v1.0.3.8, you can easily access BTCPay Server logs from the front-end. If you are a server admin, go to **Server Settings > Logs** and open the logs file. If you don't know what a particular error in the logs means, make sure to mention it when troubleshooting.
+
+If you would like more detailed logs and you're using a Docker deployment, you can view logs of specific Docker containers using the command line. See these [instructions to ssh](FAQ/FAQ-ServerSettings.md#how-to-ssh-into-my-btcpay-running-on-vps) into an instance of BTCPay running on a VPS (like LunaNode).
+
+Below is a general list of the container names used for BTCPay. Follow the lightning examples below for step by step log generation instructions.  
```

Comment:
I'd rather just post a log example here than tell them to follow a lightning example. Just post `docker logs generated_btcpayserver_1` and tell people they can change accordingly.